src/themes/components/Link/index.js

```diff
@@ -0,0 +1,43 @@
+const ComponentThemes = window.ComponentThemes;
+const { React, registerComponent, route } = ComponentThemes;
+
+// Link React component
+function Link( props ) {
+	const postInfo = {
+		id: props.postId,
+		type: props.postType,
+		slug: props.slug
+	};
+
+	return (
+		<a
+			href={ props.url }
+			className={ props.className }
+			onClick={ (e) => { route.moveTo( props.url, postInfo ); e.preventDefault(); } }
```

Comment:
I think eslint will complain about anonymous functions inside component attributes. I forget the problem but I think it has something to do with the closure leaking into the virtual dom? Anyway, easy to fix, just add `const onClick = e => route.moveTo...` above and then `onClick={ onClick }` here.